src/gitops/components/project/project-list.scss

```
@@ -1,22 +1,245 @@
+
+    th:nth-child(7),
+    td:nth-child(7) {
+      width: 4%;
+      white-space: nowrap;
+    }
+
+    // Name and description wrap instead of ellipsis truncation.
+    td:nth-child(1),
+    td:nth-child(3) {
+      white-space: normal;
+      word-break: break-word;
```

Stylelint (17.12.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)